migration/mule/foreach-processing-and-choice-routing/README.md

```diff
@@ -0,0 +1,53 @@
+# Foreach Processing and Choice Routing #
+
+This example explains how to perform content based routing along with processing per each item in the payload. 
+
+### Content Based Routing ###
+The Content-Based Router reads the content of a message and routes it to a specific recipient based on its content. It can evaluate request url, request payload or a part of it and route it to a specific path. 

```

Comment:
Shall we update to below: 

WSO2 EI has the ability to intelligently route a message through different processing pathways. Using **Switch mediator**, WSO2 EI uses an expression to evaluate a message's properties, or part of its payload, then routes it to a specific sequence (i.e. series of message mediators). Content-based routing, as this activity is called, dynamically applies routing criteria to a message at runtime.

Reply:
Done